src/ember-ops.ts

```diff
@@ -137,50 +138,95 @@ export class EmberOperation {
         });
 
+        let versionDump = exec.toString();
+
         console.log("Ember is apparently installed");
-        console.log(exec.toString());
+        console.log(versionDump);
 
-        return true;
+        return versionDump;
     } catch (e) {
         debugger;
 
-        return false;
+        return undefined;
+    }
+}
+
+var versionCache = false;
+/**
+ * Returns the versions from ember -v
+ * [0] = ember version
+ * [1] = node version
+ * [2] = os info
+ * 
+ * @returns {Array<string>} 
+ */
+function getVersionsFromDump(): Array<string> {
+    let matches;
+
+    if (!versionCache) {
+        let versionDump = getEmberVersionDump();
+        matches = versionDumpParse().exec(versionDump);
+        matches.shift();
+        versionCache = matches;
     }
+    else {
+        matches = versionCache;
+    }
+
+    return matches;
 }
 
 export function getEmberVersion(): Promise<string> {
     return new Promise((resolve, reject) => {
-        let bower;
+        let bower, bowerPath;
 
         if (!workspace || !workspace.rootPath) {
             return reject(new Error("Could not determine Ember version: Workspace not available."));
         }
 
         // Try go require the bower.json
-        try {
-            bower = require(path.join(getFullAppPath(), "bower.json"));
-        } catch (error) {
-            return reject(new Error("Could not determine Ember version: Bower.json not found."));
-        }
+        bowerPath = path.join(getFullAppPath(), "bower.json");
+        if (hasFile(bowerPath)) {
+            try {
+                bower = require(bowerPath);
+            } catch (error) {
+                return reject(new Error("Could not determine Ember version: Bower.json not found."));
+            }
 
-        // Attempt to get to the ember version
-        if (bower && bower.dependencies && bower.dependencies.ember) {
-            let version = semver().exec(bower.dependencies.ember);
+            // Attempt to get to the ember version
+            if (bower && bower.dependencies && bower.dependencies.ember) {
+                let version = semver().exec(bower.dependencies.ember);
 
-            if (version && version[0]) {
-                resolve(version[0]);
+                if (version && version[0]) {
+                    resolve(version[0]);
+                } else {
+                    return reject(new Error("Could not determine Ember version: Ember version not recognized."));
+                }
             } else {
-                return reject(new Error("Could not determine Ember version: Ember version not recognized."));
+                return reject(new Error("Could not determine Ember version: Ember not a bower dependency."));
             }
-        } else {
-            return reject(new Error("Could not determine Ember version: Ember not a bower dependency."));
+        }
+        // Get version from version dump
+        else {
+            let versions = getVersionsFromDump();
```

Comment:
If I understand everything correctly you return the Ember CLI version here, correct?  
If it is the Ember CLI version, I guess this would be wrong since the return value of this function is supposed to be the version of Ember itself and not of Ember CLI.